roomserver/version/version.go

```diff
@@ -15,84 +15,58 @@
 package version
 
 import (
-	"errors"
+	"fmt"
 
-	"github.com/matrix-org/dendrite/roomserver/state"
-)
-
-type RoomVersionID int
-type EventFormatID int
-
-const (
-	RoomVersionV1 RoomVersionID = iota + 1
-	RoomVersionV2
-	RoomVersionV3
-	RoomVersionV4
-	RoomVersionV5
-)
-
-const (
-	EventFormatV1 EventFormatID = iota + 1 // original event ID formatting
-	EventFormatV2                          // event ID is event hash
-	EventFormatV3                          // event ID is URL-safe base64 event hash
+	"github.com/matrix-org/gomatrixserverlib"
 )
 
+// RoomVersionDescription contains information about a room version,
+// namely whether it is marked as supported or stable in this server
+// version.
 type RoomVersionDescription struct {
-	Supported                 bool
-	Stable                    bool
-	StateResolution           state.StateResolutionVersion
-	EventFormat               EventFormatID
-	EnforceSigningKeyValidity bool
+	Supported bool
+	Stable    bool
```

Comment:
Please document what you actually mean by these two flags. If they refer to specific spec terms, please link to the spec.

Reply:
Will do.